README.md

```diff
@@ -329,7 +329,11 @@ It includes:
 - Debuggers
 
 ### Demo Instructions
+
+⚠️ Make sure to open the project in a Dev Container[^1] ⚠️
```

Comment:
Is it specific to an environment? I don't need any dev container

Reply:
Fair enough